### ISSUE TITLE
Setting for fixed RTP client port

### ISSUE THREAD

**Issue by Nyxtorm** [xteve-project/xTeVe#368]
Hello,

I use xteve behind a VyOS firewall where I don't manage UPnP.
On VLC on Windows, I force the RTP client port (Input / Codecs => Demuxers => RTP/RTSP => Client port) to, for example, 36002. Then I do a DNAT in UDP from ports 36001 to 36003 on my router.

But on xteve, it's not possible to force the RTP client port, it's random, so it's impossible to do a fixed DNAT for the xteve RTP flow. Would it be possible to add this option?

And maybe even allow you to define one port per tuner if you're able to use more than one? 😅

Thanks for your help!

**Comment by alturismo** [comment]
> But on xteve, it's not possible to force the RTP client port, it's random, so it's impossible to do a fixed DNAT for the xteve RTP flow. Would it be possible to add this option?
> 
> And maybe even allow you to define one port per tuner if you're able to use more than one? 😅

nope, may look at some forks if anyone did something like this ...
